catroid/src/main/java/org/catrobat/catroid/content/bricks/SetLookBrick.java

```diff
@@ -115,12 +123,13 @@ public View getView(final Context context, int brickId, BaseAdapter baseAdapter)
 
 			@Override
 			public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
-				if (position == 0) {
+				if (position == NEW_LOOK_POSITION) {
 					look = null;
 				} else {
 					look = (LookData) parent.getItemAtPosition(position);
 					oldSelectedLook = look;
 				}
+				isFormulaEnabled = (position == FORMULA_EDITOR_POSITION);
```

Comment:
maybe change the naming of the variables to `SPINNER_POSITION_NEW_LOOK` and `SPINNER_POSITION_FORMULA_EDITOR` to make it clear which position are meant